src/main/webapp/assets/js/reviews.js

```diff
@@ -20,22 +20,31 @@ async function getReviews() {
 function createReviewSection(review) {
   const reviewWrapper = document.createElement("div");
   reviewWrapper.className = "review-card";
-
+  
+  let stars = createStars(review.rating);
   reviewWrapper.innerHTML =
     `<div class="review-card-container">
       <div class="name-and-date">
         <p>${review.userID}</p>
-        <div class="rating">
-          <i class="fas fa-star yellow-star"></i>
-          <i class="fas fa-star yellow-star"></i>
-          <i class="fas fa-star yellow-star"></i>
-          <i class="fas fa-star yellow-star"></i>
-          <i class="fas fa-star yellow-star"></i>
-        </div>
+        <div class="rating">${stars}</div>
         <p>${review.dateTime}</p>
       </div>
       <p>${review.comment}</p>
     </div>`;
 
   return reviewWrapper;
 }
+
+function createStars(rating) {
+  var i=0;
+  var starHTML = '';
```

Comment:
I think `let` will be better since its within the local scope?
https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/#:~:text=var%20declarations%20are%20globally%20scoped%20or%20function%20scoped%20while%20let,be%20updated%20nor%20re%2Ddeclared.

Reply:
Oo thanks for pointing it out, I didn't know the difference haha. But after reading further seems like `var` can also be considered "local". `var` can be both global and function (local?) scope and in this case it's function scoped.
https://medium.com/edge-coders/function-scopes-and-block-scopes-in-javascript-25bbd7f293d7#:~:text=A%20variable%20declared%20with%20var,accessed%20outside%20that%20function%20scope.&text=Variables%20declared%20with%20let%20inside,loop%20index%20variable%20scope%20problem.